easybuild/easyconfigs/p/picard/picard-2.22.1-Java-11.eb

```diff
@@ -0,0 +1,50 @@
+##
+# This is an easyconfig file for EasyBuild, see https://github.com/easybuilders/easybuild
+#
+# This is a contribution from Phoenix HPC Service, The University of Adelaide, Australia
+# Homepage: https://www.adelaide.edu.au/phoenix/
+#
+# Copyright::   adelaide.edu.au/phoenix
+# Authors::	Robert Qiao <robert.qiao@adelaide.edu.au>, Exe Escobedo <exequiel.sepulvedaescobedo@adelaide.edu.au>
+# License::	MIT
+#
+# 2.10.1:
+# Adam Huffman
+# The Francis Crick Institute
+# 2.18.11:
+# Jonas Demeulemeester
+# The Francis Crick Institute
+# 2.21.1
+# Pavel Grochal (INUITS)
+##
+
+easyblock = 'JAR'
+
+name = 'picard'
+version = '2.22.1'
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = 'https://sourceforge.net/projects/picard'
```

Comment:
We should switch to https://broadinstitute.github.io/picard/  sourceforge page is dead.
Would you mind also switching the other 2 configs to use this new URL?